code/game/objects/items/defib.dm

```diff
@@ -318,8 +323,13 @@
 		return "buzzes, \"Patient's chest is obstructed. Operation aborted.\""
 
 /obj/item/shockpaddles/proc/can_revive(mob/living/carbon/human/H) //This is checked right before attempting to revive
-	if(H.stat == DEAD)
-		return "buzzes, \"Resuscitation failed - Severe neurological decay makes recovery of patient impossible. Further attempts futile.\""
+	if(world.time - H.timeofdeath > DEFIB_TIME_LIMIT)
+		return "buzzes, \"Shock delivered - Severe neurological decay detected. Further attempts futile.\""
+
+	if(H.stat == DEAD && !prob(DEFIB_REVIVE_CHANCE))
+		H.apply_damage(burn_damage_amt, DAMAGE_BURN, BP_CHEST) //We just zapped them, deal damage!
+		return "buzzes, \"Shock delivered - No heart rhythm detected. Resume CPR. \""
+	H.make_alive()
```

Comment:
You're calling this on the mob, while the proc is on the defibs, you'll want
```suggestion
	make_alive(H, H.timeofdeath)
```

Reply:
having already pushed make_alive(H), what would be the purpose of putting in (H, H.timeofdeath)?
Do forgive me that this is unironically my first ever SS13 PR. I've spent 2 years coding in C# for SS14 though, so you're totally welcome to explain in greater detail if you wish.

Reply:
ah you caught it, but it'll still need that second var to make sure it works fine

Reply:
Alright I'll trust you on this! The linter shall decide our fate. :)